###############################################################################
## System Logger  https://www.home-assistant.io/components/logger
## levels: debug, fatal, critical, error, warning, warn, info, notset
###############################################################################
default: warn
logs: !include /config/logging/include/component_log_startup.yaml
filters:
  homeassistant.components.cast.media_player:
    - 'Failed to cast media https://www.home-assistant.io/images/cast/splash.png' #ERROR (Thread-23) [homeassistant.components.cast.media_player] Failed to cast media https://www.home-assistant.io/images/cast/splash.png. Please make sure the URL is: Reachable from the cast device and either a publicly resolvable hostname or an IP address

  homeassistant.components.onvif:
    - 'Fetch snapshot image failed'

  homeassistant.components.stream:
    - 'Error from stream worker' #ERROR (stream_worker) [homeassistant.components.stream] Error from stream worker: Timestamp overflow detected: last dts = 4196969176, dts = 288552510

  homeassistant.components.nws:
    - 'Error requesting NWS observation'
    - 'Error requesting NWS forecast' #ERROR (MainThread) [homeassistant.components.nws] Error requesting NWS forecast hourly station KDTW data: 500, message='Internal Server Error', url=URL('https://api.weather.gov/gridpoints/DTX/68,35/forecast/hourly')

  pybalboa.balboa:
    - 'Spa stopped responding, requesting panel config.' #ERROR (MainThread) [pybalboa.balboa] Spa stopped responding, requesting panel config.
    - 'Lost connection to spa, attempting reconnect' #ERROR (MainThread) [pybalboa.balboa] Lost connection to spa, attempting reconnect.
    - 'Error connecting to spa' #ERROR (MainThread) [pybalboa.balboa] Error connecting to spa at 192.168.1.125:4257: [Errno 110] Connect call failed ('192.168.1.125', 4257)
    - 'Connection reset by peer' #ERROR (MainThread) [pybalboa.balboa] Connection reset by peer
    - 'Message had bad CRC' #ERROR (MainThread) [pybalboa.balboa] Message had bad CRC, discarding

  homeassistant.components.http.ban:
    - 'Login attempt or request with invalid authentication' #WARNING (MainThread) [homeassistant.components.http.ban] Login attempt or request with invalid authentication from 192.168.1.60 (192.168.1.60). (Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.45 Safari/537.36)

  homeassistant.helpers.entity:
    - 'Update for sensor.chromecast_devices fails' #ERROR (MainThread) [homeassistant.helpers.entity] Update for sensor.chromecast_devices fails
    - 'is taking over 10 seconds' #WARNING (MainThread) [homeassistant.helpers.entity] Update of binary_sensor.wan_connected is taking over 10 seconds
    - 'took longer than the scheduled update interval' #WARNING (MainThread) [homeassistant.components.sensor] Updating breaking_changes sensor took longer than the scheduled update interval 0:00:05

  pylutron_caseta.smartbridge:
    - 'ping was not answered. closing connection.' #WARNING (MainThread) [pylutron_caseta.smartbridge] ping was not answered. closing connection.
    - 'LEAP session ended. Reconnecting' #WARNING (MainThread) [pylutron_caseta.smartbridge] LEAP session ended. Reconnecting...
    - 'Reconnecting...' #WARNING (MainThread) [pylutron_caseta.smartbridge] Reconnecting...

  pychromecast.socket_client:
    - 'Heartbeat timeout, resetting connection' #WARNING (Thread-24) [pychromecast.socket_client] [Living Room Speaker(192.168.1.198):8009] Heartbeat timeout, resetting connection
    - 'Failed to connect to service' #ERROR (Thread-22) [pychromecast.socket_client] [Broadcast Speakers([::ffff:c0a8:1c3]):32169] Failed to connect to service ServiceInfo(type='host', data=('[::ffff:c0a8:1c3]', 32169)), retrying in 5.0s
    - 'Error reading from socket' #ERROR (Thread-22) [pychromecast.socket_client] [Broadcast Speakers(192.168.1.194):32169] Error reading from socket.
    - 'Error communicating with socket' #WARNING (Thread-22) [pychromecast.socket_client] [Broadcast Speakers(192.168.1.194):32169] Error communicating with socket, resetting connection

  homeassistant.components.mold_indicator.sensor:
    - 'Unable to parse' #ERROR (MainThread) [homeassistant.components.mold_indicator.sensor] Unable to parse temperature sensor sensor.outdoor_temperature with state: unavailable

  homeassistant.components.input_select:
    - 'no longer valid' #WARNING (MainThread) [homeassistant.components.input_select] Current option: none no longer valid (possible options: Oldies but Goldies, Classic Oldies, All About Country,

  homeassistant.components.sensor:
    - 'took longer than the scheduled update interval' #WARNING (MainThread) [homeassistant.components.sensor] Updating breaking_changes sensor took longer than the scheduled update interval 0:00:05
    - 'Setup of sensor' #WARNING (MainThread) [homeassistant.components.sensor] Setup of sensor platform darksky is taking over 10 seconds.
    - 'waqi: Error on device update!' #ERROR (MainThread) [homeassistant.components.sensor] waqi: Error on device update!

  homeassistant.components.media_player:
    - 'Setup of' #WARNING (MainThread) [homeassistant.components.media_player] Setup of media_player platform roku is taking over 10 seconds.
    - 'took longer than the scheduled update interval' #WARNING (MainThread) [homeassistant.components.media_player] Updating webostv media_player took longer than the scheduled update interval 0:00:10
  homeassistant.components.binary_sensor:
    - 'Setup of' #WARNING (MainThread) [homeassistant.components.binary_sensor] Setup of binary_sensor platform pi_hole is taking over 10 seconds.
    - 'took longer than the scheduled update interval' #WARNING (MainThread) [homeassistant.components.binary_sensor] Updating hydrawise binary_sensor took longer than the scheduled update interval 0:00:30

  homeassistant.setup:
    - 'is taking over 10 seconds' #WARNING (MainThread) [homeassistant.setup] Setup of group is taking over 10 seconds.

  homeassistant.components.weather:
    - 'is taking over 10 seconds' #WARNING (MainThread) [homeassistant.components.weather] Setup of weather platform template is taking over 10 seconds.

  homeassistant.components.light:
    - 'is taking over 10 seconds' #WARNING (MainThread) [homeassistant.components.light] Setup of light platform virtual is taking over 10 seconds.

  homeassistant.components.fan:
    - 'is taking over 10 seconds' #WARNING (MainThread) [homeassistant.components.fan] Setup of fan platform template is taking over 10 seconds.

  homeassistant.components.switch:
    - 'is taking over 10 seconds' #WARNING (MainThread) [homeassistant.components.switch] Setup of switch platform template is taking over 10 seconds.
    - 'took longer than the scheduled update interval' #WARNING (MainThread) [homeassistant.components.switch] Updating hydrawise switch took longer than the scheduled update interval 0:00:30
  homeassistant.components.geo_location:
    - 'is taking over 10 seconds' #WARNING (MainThread) [homeassistant.components.geo_location] Setup of geo_location platform gdacs is taking over 10 seconds.
  homeassistant.config_entries:
    - 'integration not ready yet' #WARNING (MainThread) [homeassistant.config_entries] Config entry 'Home' for accuweather integration not ready yet; Retrying in background

  homeassistant.components.updater:
    - 'Timeout fetching Home Assistant update data' #ERROR (MainThread) [homeassistant.components.updater] Timeout fetching Home Assistant update data
    - 'Error requesting Home Assistant update data' #ERROR (MainThread) [homeassistant.components.updater] Error requesting Home Assistant update data: [Errno 32] Broken pipe
  homeassistant.components.calendar:
    - 'is taking over 10 seconds' #WARNING (MainThread) [homeassistant.components.calendar] Setup of calendar platform google is taking over 10 seconds.

  homeassistant.components.camera:
    - 'is taking over 10 seconds' #WARNING (MainThread) [homeassistant.components.camera] Setup of camera platform generic is taking over 10 seconds.

  homeassistant.components.scene:
    - 'is taking over 10 seconds' #WARNING (MainThread) [homeassistant.components.scene] Setup of scene platform homeassistant is taking over 10 seconds.
  aio_georss_client.feed:
    - 'failed with timeout error' #Requesting data from https://www.gdacs.org/xml/rss.xml failed with timeout error

  aio_georss_client.feed_manager:
    - 'Update not successful' #WARNING (MainThread) [aio_georss_client.feed_manager] Update not successful, no data received from <GdacsFeed(home=(42.30427713978584, -82.92516946792604), url=https://www.gdacs.org/xml/rss.xml, radius=10000, categories=[])>

  homeassistant.components.mqtt:
    - 'Disconnected from MQTT server' #WARNING (Thread-5) [homeassistant.components.mqtt] Disconnected from MQTT server core-mosquitto:1883 (7)

  hass_nabucasa.cloud_api:
    - 'Fetched https://remotestate.nabucasa.com/request_sync' #WARNING (MainThread) [hass_nabucasa.cloud_api] Fetched https://remotestate.nabucasa.com/request_sync (500)

  hole:
    - 'Can not load data from' #ERROR (MainThread) [hole] Can not load data from *hole: 192.168.1.110:80

  homeassistant.components.hassio.handler:
    - 'Timeout on' #ERROR (MainThread) [homeassistant.components.hassio.handler] Timeout on /addons/core_zwave_js/info request

  homeassistant.components.zwave_js:
    - 'Failed to get the Z-Wave JS add-on info' #ERROR (MainThread) [homeassistant.components.zwave_js] Failed to get the Z-Wave JS add-on info:

  aioasuswrt.connection:
    - 'Host timeout' #ERROR (MainThread) [aioasuswrt.connection] Host timeout.

  homeassistant.bootstrap:
    - 'Waiting on integrations to complete setup' #WARNING (MainThread) [homeassistant.bootstrap] Waiting on integrations to complete setup: sensor.streaming_radio_player, sensor.history_stats, sensor.mold_indicator, sensor.statistics, sensor.integration, sensor.sun2, sensor.dnsip, sensor.command_line, sensor.hydrawise,

  matplotlib.font_manager:
    - 'Matplotlib is building the font cache' #WARNING (Thread-49) [matplotlib.font_manager] Matplotlib is building the font cache; this may take a moment.

  homeassistant.components.roku.coordinator:
    - 'Error fetching roku data' #ERROR (MainThread) [homeassistant.components.roku.coordinator] Error fetching roku data: Invalid response from API: Timeout occurred while connecting to device

  homeassistant.components.pi_hole:
    - 'Error fetching Pi-Hole data' #ERROR (MainThread) [homeassistant.components.pi_hole] Error fetching Pi-Hole data: Failed to communicate with API: Can not load data from *hole: 192.168.1.110:80

  homeassistant.components.waqi.sensor:
    - 'Failed to connect to WAQI servers' #ERROR (MainThread) [homeassistant.components.waqi.sensor] Failed to connect to WAQI servers

  custom_components.authenticated.sensor:
    - 'New successful login' #WARNING (SyncWorker_4) [custom_components.authenticated.sensor] New successful login from unknown IP (192.168.1.231)
